config/locales/de.yml

```diff
@@ -150,6 +150,7 @@ de:
         conversation: Ich wünsche ein Gespräch mit meiner/meinem Freiwilligenverantwortlichen.
         future: Soll der Einsatz weiterlaufen und wenn ja, mit welchen Inhalten (Zielen)?
         goals: Was waren die wichtigsten Inhalte (oder Ziele) Ihres Einsatzes in den letzten Monaten?
+        last: Letzter inhaltliches Journal Eintrag am
```

Comment:
"Letzter inhaltlicher Journaleintrag am"

Not sure about the wording though, what do you think @nuritnt @casaper?

Reply:
there will anyway be a renaming for the two different journals, but we are waiting back for feedback from aoz how to rename it, so for the time being for me its fine like that